integration/ct_integration_test.go

```diff
@@ -66,16 +66,16 @@ func TestCTIntegration(t *testing.T) {
 	// Stage 0: get accepted roots, which should just be the fake CA.
 	roots, err := logClient.GetAcceptedRoots(ctx)
 	if err != nil {
-		t.Fatalf("Failed to get roots: %v", err)
+		t.Fatalf("GetAcceptedRoots()=(nil,%v); want (roots,nil)", err)
```

Comment:
Again, can I leave till a later pass so the two branches on top of this week-old pull request don't need more rebasing?